### PR TITLE
Remove console deployments from list of 2 replicas in single-replica topology

### PR DESCRIPTION
**Pull request by jhadvig**
Based on https://github.com/openshift/console-operator/pull/508 change.

/assign @romfreiman

### CODE REVIEW

File: test/extended/single_node/topology.go [hunk 1]
```diff
@@ -11,7 +13,6 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	e2e "k8s.io/kubernetes/test/e2e/framework"
 	e2eskipper "k8s.io/kubernetes/test/e2e/framework/skipper"
-	"strings"
```
Comment (romfreiman) [inline test/extended/single_node/topology.go:14]: ?

Comment (jhadvig) [reply test/extended/single_node/topology.go:14]: Was fixed by my IDE, since its ordering of the imported pkgs is wrong here

**Comment by omertuc** [comment]
strange failure https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/25929/pull-ci-openshift-origin-master-e2e-aws-single-node/1369619750568595456

/test e2e-aws-single-node

**Comment by romfreiman** [comment]
@omertuc @jhadvig seemst that the relevant part passed. We can merge.

**Comment by deads2k** [comment]
/approve

the hold isn't mine, but this looks ok to me.

**Pull request merged**